src/components/DocumentationTopic/TopicsLinkBlock.vue

```diff
@@ -179,6 +180,7 @@ export default {
       if (topic.titleStyle === TitleStyles.title) {
         return topic.ideTitle ? 'span' : 'code';
       }
+      if (topic.role && topic.role === TopicRole.collection) return 'span';
```

Comment:
Maybe add a comment, explaining why we do this?

Reply:
Got it, will do